src/object/srv_obj_remote.c

```diff
@@ -408,12 +408,17 @@ ds_obj_cpd_dispatch(struct dtx_leader_handle *dlh, void *arg, int idx,
 	struct daos_cpd_sg		*head_dcs = NULL;
 	struct daos_cpd_sg		*dcsr_dcs = NULL;
 	struct daos_cpd_sg		*dcde_dcs = NULL;
+	uint64_t			 time1 = 0;
+	uint64_t			 time2 = 0;
+	uint32_t			 diff;
 	int				 total;
 	int				 count;
 	int				 rc = 0;
 
 	D_ASSERT(idx < dlh->dlh_sub_cnt);
 
+	sched_exec_time(&time1, NULL);
```

Comment:
Is this for debug only, or you wish it being landed to master? The intention of using 'sched_exec_time()' instead of using "daos_getmtime_corase()" is that you don't want it enabled by default?

Reply:
This is just for debug, will not be landed to master.